cshsms/settings.py

```diff
@@ -16,7 +16,7 @@
 
 
 if not os.getenv('IS_TRAVIS', False):
-    from cshsms.settings_secret import SECRET_KEY, DATABASES, REMOTE
+    from cshsms.settings_secret import SECRET_KEY, DATABASES, REMOTE, TEXTLOCAL_API, TEXTLOCAL_PRIMARY_ID
 else:
     from cshsms.settings_travis import SECRET_KEY, DATABASES, REMOTE
```

Comment:
That is, what you had before was right, but failed because you don't assign the variables within `settings_travis` like you need to.